### ISSUE TITLE
Addition to the /chrysanthemum-test command

### ISSUE THREAD

**Issue by ValiantWind** [AmaranthineCodices/chrysanthemum#33]
Currently, the /chrysanthemum-test command only displays whether it gets filtered or not:
https://cdn.discordapp.com/attachments/1028104917170258030/1064342427541979156/image.png

My suggestion would be so it tells us **which filter** its under.

By that I mean, in addition to showing us of what filtered it failed to pass, tell us which level filter it falls under (i.e. is it below level 2, below level 10, or general filter) and the punishment for it (i.e. is it an automatic mute for 1 hour or not)

**Examples:**

If we enter in the word "stfu" like in the image link below, it will **also** tell us that that its an "Automatic Mute (1 hour)
https://cdn.discordapp.com/attachments/1028104917170258030/1064340841503674408/image.png

Or if we type in "dm me" like in the image link below, it will **also** say that its in the "Below Level 10 filter"
https://cdn.discordapp.com/attachments/1028104917170258030/1064341145431314522/image.png

Basically what shows up in #censored-and-spam-logs under **Filter** when a message gets filtered.
